File: app/src/main/java/moe/kotohana/randomfood/models/Place.java
package moe.kotohana.randomfood.models;

import java.io.Serializable;

/**
 * Created by Junseok Oh on 2017-06-13.
 */

public class Place implements Serializable {
    private String name;

    public Place(String name) {
        this.name = name;
    }

    public String getName() {
        return name;
    }
}

File: app/src/main/java/moe/kotohana/randomfood/models/Place.kt
package moe.kotohana.randomfood.models

import android.util.Log
import io.realm.RealmObject
import java.io.Serializable

/**
 * Created by Junseok Oh on 2017-06-13.
 */

open class Place(val items: ArrayList<Restaurant>) : Serializable
